lib/labels/github/api.ex

```diff
@@ -53,6 +53,9 @@ defmodule Labels.Github.Api do
   end
 
   defp check_api_response(res) do
+    if res.status != 200 and res.status != 201,
+      do: Logger.info("#{res.status}: #{res.body["message"]}")
```

Comment:
Good plan. 🔍 👍

Reply:
This PR is already deployed on Fly if you want to try again, I'm monitoring the logs